From 020a52d6c783959f31024649877d40f0a1ed24d1 Mon Sep 17 00:00:00 2001
From: Ryan Blue <blue@apache.org>
Date: Mon, 28 Sep 2015 12:57:25 -0400
Subject: [PATCH 113/140] KITE-1083: Add warnings to POM files about Hive
 changes.

This warns that the jars needed to connect to the metastore are pulled
in individually and kite-tools should be checked when Hive dependencies
change to avoid regressions.
---
 kite-tools-parent/pom.xml |    5 +++++
 pom.xml                   |    6 ++++++
 2 files changed, 11 insertions(+)

diff --git a/kite-tools-parent/pom.xml b/kite-tools-parent/pom.xml
index e1b32b8..a49ce5f 100644
--- a/kite-tools-parent/pom.xml
+++ b/kite-tools-parent/pom.xml
@@ -207,6 +207,11 @@
          in the dependency modules
          -->
 
+    <!-- When changing Hive dependencies, be careful that the required jars are
+         pulled in by the TransformTask. New dependencies should be added to
+         the distributed cache by class.
+         -->
+
     <dependency>
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-exec</artifactId>
diff --git a/pom.xml b/pom.xml
index 5865bc8..b24e6d1 100644
--- a/pom.xml
+++ b/pom.xml
@@ -784,6 +784,12 @@
       </dependency>
 
       <!-- Hive dependencies -->
+
+      <!-- When changing Hive dependencies, be careful that the required jars are
+           pulled in by the TransformTask in kite-tools. New dependencies
+           should be added to the distributed cache by class.
+           -->
+
       <dependency>
         <groupId>org.apache.hive</groupId>
         <artifactId>hive-exec</artifactId>
-- 
1.7.9.5

